app/javascript/controllers/sessionstorage_amend_form_controller.js

```diff
@@ -26,6 +26,10 @@ export default class extends Controller {
         this.fieldTarget.appendChild(element);
       }
     }
+    // Adds the selection count
+    if (this.hasCountTarget) {
+      this.countTarget.innerText = `${this.countTarget.id} ${storageValues.length}`
```

Comment:
I'm not a huge fan of hijacking a component that is used other places to do a single piece of new UI.  It would probably be better to create a export controller that updates the UI.  Also, please never use the id as a way to get a custom string, a data attribute would be a much better choice. Try creating a summary section before your dropdown that says something like: "You are about to export X samples from this project..." and explain briefly that they will be made into a zip file and you will be redirected to the download page (or something like that).  Sorry this part should be on the html, but wanted to keep both comments together

Reply:
Added a summary and a `data_export_controller` to grab the sample count. 8cfbfce670dc7830e1675947b000d13be9846edb